organization/_posts/2019/2019-02-28-coordination.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "HSF Weekly Meeting #160, 28 February, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+*Present/Contributors*: Graeme Stewart, André Sailer, Marko Petric,
+Tommaso Boccali, Danilo Piparo, Pere Mato, Stefan Roiser, Andrea
+Valassi, Eduardo Rodrigues, David Lange, Agnieszka Dziurda, Serhan Mete,
+Andrea Rizzi, Caterina Doglioni, Ed Moyse, Jim Amundson, Liz
+Sexton-Kennedy, Mark Hodgkinson, Daniel Elvira, Heather Grey, Markus
```

Comment:
Noticed by change - I believe it's actually "Gray" ;-). To be checked.

Reply:
Gloria Corti to be added.

Reply:
You were right on both counts - thanks!